test/e2e/velero_utils.go

```diff
@@ -4,13 +4,14 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
+	"github.com/pkg/errors"
```

Comment:
I think you'll need to run `make update` to sort these imports.

Reply:
Yup, done